docs/modules/operation/pages/events/event-configuration.adoc

```diff
@@ -1,39 +1,47 @@
 
 [[ga-events-event-configuration]]
-= Event Configuration
+= Event Daemon Configuration
 
-The back-end configuration surrounding events is broken into two areas: the configuration of `Eventd` itself, and the configuration of all types of events known to {page-component-title}.
+The back-end configuration surrounding events is split into two areas: the configuration of Eventd itself, and the configuration of events definitions known to {page-component-title}.
 
 == The eventd-configuration.xml file
 
-The overall behavior of `Eventd` is configured in the file `$\{OPENNMS_HOME}/etc/eventd-configuration.xml`.
+The overall behavior of Eventd is configured in the `$\{OPENNMS_HOME}/etc/eventd-configuration.xml` file.
 This file does not need to be changed in most installations.
-The configurable items include:
+The configurable items include the following:
 
 TCPAddress::
-    The IP address to which the `Eventd` XML/TCP listener will bind. Defaults to `127.0.0.1`.
+    The IP address to which the Eventd XML/TCP listener will bind.
+    Defaults to `127.0.0.1`.
 TCPPort::
-    The TCP port number on `TCPAddress` to which the `Eventd` XML/TCP listener will bind. Defaults to `5817`.
+    The TCP port number on `TCPAddress` to which the Eventd XML/TCP listener will bind.
+    Defaults to `5817`.
 UDPAddress::
-    The IP address to which the `Eventd` XML/UDP listener will bind. Defaults to `127.0.0.1`.
+    The IP address to which the Eventd XML/UDP listener will bind.
+    Defaults to `127.0.0.1`.
 UDPPort::
-    The UDP port number on `TCPAddress` to which the `Eventd` XML/UDP listener will bind. Defaults to `5817`.
+    The UDP port number on `TCPAddress` to which the Eventd XML/UDP listener will bind.
+    Defaults to `5817`.
 receivers::
-    The number of threads allocated to service the event intake work done by `Eventd`.
+    The number of threads allocated to service the event intake work done by Eventd.
 queueLength::
-    The maximum number of events that may be queued for processing. Additional events will be dropped. Defaults to unlimited.
+    The maximum number of events that may be queued for processing.
+    Additional events will be dropped.
+    Defaults to unlimited.
 getNextEventID::
-    An SQL query statement used to retrieve the ID of the next new event. Changing this setting is not recommended.
+    An SQL query statement used to retrieve the ID of the next new event.
+    Changing this setting is not recommended.
```

Comment:
It would depend on how severe the consequences would be. I'd like to minimize the use of admonitions. @mmahacek do you know how serious it is if they did change the setting?

Reply:
I don't recall ever changing this setting.